src/main/java/baseball/View/OutputView.java

```diff
@@ -0,0 +1,46 @@
+package baseball.View;
+
+import baseball.Constant.BallStatus;
+import java.util.Map;
+
+public class OutputView {
+
+  public static final String START_GAME_MESSAGE = "숫자 야구 게임을 시작합니다.";
+  public static final String GAME_RESULT_MESSAGE = "\n%d개의 숫자를 모두 맞히셨습니다! 게임 종료";
+  public static final String END_GAME_MESSAGE = "게임 종료";
+  public static final int MAX_STRIKE_COUNT = 3;
+
+  public static void printGameStart() {
+    System.out.println(START_GAME_MESSAGE);
+  }
+
+  public static void printBallsStatus(Map<BallStatus, Integer> scoreCount) {
+    boolean nothingCondition = isNothing(scoreCount);
+
+    if (nothingCondition) {
+      System.out.printf("%s", BallStatus.NOTHING.getStatus()); // 낫싱
+    }
+    scoreCount.forEach((status, count) -> {
+      if (!nothingCondition && count > 0) { // 스트라이크와 볼의 개수가 0보다 크면 실행
+        System.out.printf("%d%s ", count, status.getStatus()); // %d개 스트라이크(볼)
```

Comment:
```suggestion
  public static void printBallsStatus(Map<BallStatus, Integer> scoreCount) {
    if (isNothing(scoreCount)) {
      System.out.printf("%s", BallStatus.NOTHING.getStatus()); // 낫싱
      return;
    }
    scoreCount.forEach((status, count) -> {
      if (count > 0) { // 스트라이크와 볼의 개수가 0보다 크면 실행
        System.out.printf("%d%s ", count, status.getStatus()); // %d개 스트라이크(볼)
```

사소하지만 nothingCondition은 맨 위 분기에서만 사용되서 없앨 수 있을 것 같습니다!

Reply:
아! 두번 체킹되는군요! else를 사용하지 않고 코드를 짜는 요구사항을 과도하게 신경쓰다 놓쳤네요 하하하 검증 조건도 nothingCondition같이 따로 분리해서 메소드로 구현하면 가독성도 더 좋아질 것 같아요!